corehq/apps/fixtures/resources/v0_1.py

```diff
@@ -204,10 +204,10 @@ def obj_delete(self, bundle, **kwargs):
         return ImmediateHttpResponse(response=HttpAccepted())
 
     def obj_create(self, bundle, request=None, **kwargs):
-        if 'data_type_id' not in bundle.data:
-            raise BadRequest("data_type_id must be specified")
+        data_type_id = bundle.data.get('data_type_id', None)
 
-        data_type_id = bundle.data['data_type_id']
+        if not data_type_id:
```

Comment:
Is it certain `data_type_id` will never evaluate to false? Might be safer to do `data_type_id is None`

Reply:
In the current error, ```data_type_id == ""```